save-backend/src/main/kotlin/org/cqfn/save/backend/repository/TimestampBasedFileSystemRepository.kt

```diff
@@ -104,6 +126,44 @@ class TimestampBasedFileSystemRepository(configProperties: ConfigProperties) {
             }
     }
 
+    /**
+     * @param owner owner name
+     * @param part file part
+     * @return Mono with number of bytes saved
+     * @throws FileAlreadyExistsException if file with this name already exists
+     */
+    fun saveImage(part: Mono<FilePart>, owner: String): Mono<ImageInfo> = part.flatMap { part ->
```

Comment:
It's the same as `saveFile` and only differs in directory name. Please reduce code duplication

Reply:
Moved common logic to a new method